### PR TITLE
API namespace CMS

### PR DESCRIPTION
**Pull request by tractorcow**
See https://github.com/silverstripe/silverstripe-cms/issues/1572

### CODE REVIEW

File: code/Controllers/CMSMain.php [hunk 1]
```diff
@@ -403,16 +414,18 @@ public function SearchForm() {
...
 		foreach($actions->dataFields() as $action) {
+			/** @var FormAction $action */
 			$action->setUseButtonTag(true);
 		}
 
 		// Create the form
-		$form = Form::create($this, 'SilverStripe\\CMS\\Search\\SearchForm', $fields, $actions)
+		/** @skipUpgrade */
+		$form = Form::create($this, 'SearchForm', $fields, $actions)
 			->addExtraClass('cms-search-form')
```
Comment (hafriedlander) [inline code/Controllers/CMSMain.php]: This wasn't required before. How is this change related to namespacing?

Comment (tractorcow) [inline code/Controllers/CMSMain.php]: I think it came out of a post-namespace test; It may have been a pre-existing issue I fixed here.

**Comment by hafriedlander** [comment]
Failures are in psql & sqlite. Merging to avoid even-more-broken master, but need fixes for those modules ASAP.
